update.sh

```diff
@@ -143,6 +146,8 @@ print_basepackages() {
 	    unzip \
 	    wget \
 	    zip && \
+	    apt-get clean && \
+	    rm -rf /var/lib/apt/lists/* && \
```

Comment:
I think these lines are unnecessary because they are already part of `print_cleanup`. Doing the cleanup in `print_basepackages` can cause issues when packages are installed in other steps. Or it can make the build slower because the package info has to be updated again.